crates/cli/tests/upgrade_test.rs

```diff
@@ -1,8 +1,5 @@
 mod utils;
 
-use proto_shim::get_exe_file_name;
```

Comment:
These are probably used by the `unix` cfg.

Reply:
Revert back this change, you are right. I am developing on windows and the linter removed this auto.